src/linker/Linker.Steps/UnreachableBlocksOptimizer.cs

```diff
@@ -10,12 +10,13 @@
 namespace Mono.Linker.Steps
```

Comment:
Should it move to the Mono.Linker namespace and directory?

Reply:
I'd keep it inside Steps

Reply:
I thought about it - it's basically now part of MarkStep (in a way)... so I would like to keep it in the Steps.